src/Propulsion.Kafka/Consumers.fs

```diff
@@ -3,7 +3,11 @@
 namespace Propulsion.Kafka
 
 open Confluent.Kafka
+#if CONFLUENT_KAFKA_SHIMS
```

Comment:
Perhaps it makes more sense for the symbol to also be `KAFKA0` here given it controls both CK and JCKFS shimming?

Reply:
👍 yes, it simplify things. done!

Reply:
looks great - will merge tomorrow (if we don't find a cleaner way to address the conflict)